src/lib/components/Footer.svelte

```diff
@@ -0,0 +1,151 @@
+<script>
+  import Container from "$lib/components/Container.svelte"
+  import Grid from "$lib/components/Grid.svelte"
+  import Button from "$lib/components/Button.svelte"
+
+  import IOXIO from "$lib/images/ioxio-logo.svg?dataurl"
+  import mail from "$lib/images/mail-icon.svg"
+  import slack from "$lib/images/slack-icon.svg"
+  import arrow from "$lib/images/link-arrow-icon.svg?dataurl"
+
+  const SLACK_URL = "https://slack.ioxio.com"
+  const EMAIL_URL = "mailto:support@ioxio.com"
+</script>
+
+<footer>
+  <div class="top-section-wrapper">
+    <Container>
+      <Grid container class="footer-container">
+        <Grid sm={0} lg={1} />
+        <Grid sm={12} md={6}>
+          <div class="try-out-section">
+            <h3>Try out our technologies for free on</h3>
+            <div class="link">
+              <a href="https://sandbox.ioxio-dataspace.com"> IOXIO Dataspace&trade; Sandbox </a>
+              <img src={arrow} alt="arrow" />
+            </div>
+          </div>
+        </Grid>
+        <Grid sm={0} lg={5} />
+      </Grid>
+    </Container>
+  </div>
+
+  <div class="bottom-section-wrapper">
+    <Container>
+      <Grid container>
+        <Grid sm={0} lg={1} />
+        <Grid sm={12} md={9} lg={8}>
+          <div class="contact-us-wrapper">
+            <h3>Contact us by using the available support options</h3>
+            <div class="contact-us-buttons">
+              <Button icon={slack} url={SLACK_URL}>Slack</Button>
+              <Button icon={mail} monochromeIcon url={EMAIL_URL}>Email</Button>
+            </div>
+          </div>
+        </Grid>
+        <Grid sm={12} md={3} lg={2} class="ioxio-logo-grid-item">
+          <img class="ioxio-logo" src={IOXIO} alt="IOXIO" />
```

Comment:
I'm not sure reading this out loud adds value either. On the top navbar it does make sense to say "IOXIO" "Documentation portal" .. but here I don't think so?

Reply:
for some reason if I use an SVG there instead of img, it doesn't work well (the image is cut off) and I can't quickly find a way to fix it. I suggest we keep it there for now, removing alt property will result in a warning

Reply:
you should likely use `aria-hidden="true"` if you remove alt, but this isn't something we want a screen reader to read